CONTRIBUTING.md

```diff
@@ -1,3 +1,10 @@
 # Contributing
 
-Please have a look at our [documentation site](https://subnauticanitrox.github.io/Documentation/) for contribution guidelines & information on how to get started.
\ No newline at end of file
+Please have a look at our [documentation site](https://subnauticanitrox.github.io/Documentation/) for contribution guidelines & information on how to get started.
+
+## Stand on AI and generated issues & prs
+
+We generally don't want to tell you what tools to use (you undoubting minimalistic vi enjoyer), but we ideally want to have clean and minimal code, issues and pull requests. \
+For this reason we are ceasing the creation of AI generated issue and pr descriptions. We don't have the resources to read 50 paragraphs on why changing 5 lines of code makes sense. Only explain yourself if the goal your implementation is not a no-brainer. \
```

Comment:
```suggestion
For this reason we would prefer AI is not used in the creation of issues and pr descriptions. We don't have the resources to read 50 paragraphs on why changing 5 lines of code makes sense. Only explain yourself if the goal your implementation is not a no-brainer. \
```
I think that it might change this slightly to not be as absolute as ceasing, I do completly agree with this just thought slightly softer is better for not scaring people off the project even if they don't use AI